sotodlib/hwp/update_hwp_angle.py

```diff
@@ -0,0 +1,517 @@
+import os
+import numpy as np
+import scipy.interpolate
+import so3g
+from spt3g import core
+import argparse
+
+class update_hwp_angle(): 
+    
+    def __init__(self, debug=False):
+
+        """
+        Class to manage a L2 HK data into HWP angle g3.
+
+        Args
+        -----
+            debug: bool
+                bit for dubug mode
+        """
+
+        
+        self._debug = debug
+        self._start = 0
+        self._end = 0
+        # Size of pakcets sent from the BBB
+        self._pkt_size = 120 # maybe 120 in the latest version, 150 in the previous version
+        # Number of encoder slits per HWP revolution
```

Comment:
these seem like a hardware config things. Could we add them to a HWP config file so we can be prepared for it to change with time/device (even if we have no plans of changing things, SO lasts a long time and will have many HWPs). 

This config could include the fields names I comment on below.

Reply:
Same as above. Implemented config.